src/Microsoft.DotNet.Build.Tasks/PackageFiles/tests.targets

```diff
@@ -86,6 +100,8 @@
 
     <XunitOptions Condition="'$(Performance)'!='true'">$(XunitOptions) -notrait Benchmark=true</XunitOptions>
 
+    <XunitOptions Condition="'$(UseDotNetNativeToolchain)'=='true'">$(XunitOptions) -redirectoutput</XunitOptions>    
```

Comment:
Used along with Xunit.console.netcore changes to get the console output into a txt file after execution.